### ISSUE TITLE
** ERROR: directory is already being watched! **

### ISSUE THREAD

**Issue by gr1d99** [rails/rails#34912]
Someone help me get rid of this errors

### Steps to reproduce

- create a new rails api only application `rails new sample_api --api`
- install `create-react-app`
- create a new react app in the root directory of the new rails api app `create-react-app client`
- start rails app `bin/rails s`

### Expected behavior

```
| => bin/rails s
=> Booting Puma
=> Rails 5.2.2 application starting in development 
=> Run `rails server -h` for more startup options
Puma starting in single mode...
* Version 3.12.0 (ruby 2.5.1-p57), codename: Llamas in Pajamas
* Min threads: 5, max threads: 5
* Environment: development
* Listening on tcp://0.0.0.0:3000
Use Ctrl-C to stop
```

### Actual behavior

```
| => bin/rails s
=> Booting Puma
=> Rails 5.2.2 application starting in development 
=> Run `rails server -h` for more startup options
Puma starting in single mode...
* Version 3.12.0 (ruby 2.5.1-p57), codename: Llamas in Pajamas
* Min threads: 5, max threads: 5
* Environment: development
* Listening on tcp://0.0.0.0:3000
Use Ctrl-C to stop
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/.bin/terser-webpack-plugin

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/terser-webpack-plugin

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/.bin/url-loader

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/url-loader

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/.bin/mini-css-extract-plugin

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/mini-css-extract-plugin

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/.bin/file-loader

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/file-loader

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/react-scripts/node_modules/.bin/terser-webpack-plugin

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/terser-webpack-plugin

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/react-scripts/node_modules/.bin/url-loader

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/url-loader

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/react-scripts/node_modules/.bin/mini-css-extract-plugin

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/mini-css-extract-plugin

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/react-scripts/node_modules/.bin/file-loader

        is already being watched through: /Users/gr1d99/Projects/ruby/rails/untitled3/client/node_modules/file-loader

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors

```

### System configuration
**Rails version**: 5.2.2

**Ruby version**: 2.5.1-p57

**OS** macOS High Sierra version 10.13.2

**Comment by tabishiqbal** [comment]
Please see this thread.  https://github.com/rails/rails/issues/32700

**Comment by y-yagi** [comment]
Thanks for your report. This already fixed by #33822. Please see https://github.com/rails/rails/issues/32700#issuecomment-417645788 for a temporary solution.

**Comment by rolandoalvarado** [comment]
try this, it solves my problem. https://github.com/guard/listen/wiki/Duplicate-directory-errors
